node/resolvers/checkout.ts

```diff
@@ -0,0 +1,20 @@
+export const nearestPickupPoints = async (
+  _: any,
+  { lat, long }: CheckoutPickupPointsArgs,
+  ctx: Context
+) => {
+  const {
+    clients: { checkout },
+  } = ctx
+
+  try {
+    const closestPickupPointsData = await checkout.getNearestPickupPoints(
+      lat,
+      long
+    )
+
+    return closestPickupPointsData
+  } catch (error) {
+    console.log(error.response, '----specific errror----')
```

Comment:
We should remove the try / catch here. The way it's now, if something goes wrong, we will not pass the error to graphql.